config/config.go

```diff
@@ -449,6 +450,7 @@ func NewConfig() *Config {
 	flag.StringVar(&cfg.KubernetesRedisServiceName, "kubernetes-redis-service-name", "", "Sets name for redis to be used to lookup endpoints")
 	flag.IntVar(&cfg.KubernetesRedisServicePort, "kubernetes-redis-service-port", 6379, "Sets the port for redis to be used to lookup endpoints")
 	flag.StringVar(&cfg.KubernetesBackendTrafficAlgorithmString, "kubernetes-backend-traffic-algorithm", kubernetes.TrafficPredicateAlgorithm.String(), "sets the algorithm to be used for traffic splitting between backends: traffic-predicate or traffic-segment-predicate")
+	flag.StringVar(&cfg.KubernetesDefaultLoadBalancerAlgorithm, "kubernetes-default-lb-algorithm", "roundRobin", "sets the default loadbalancer algorithm to be used for traffic splitting between backends: roundrobin, random, consistentHash, powerOfRandomNChoices. default: roundRobin")
```

Comment:
this only applies as default for binary users. We need also check if the kubernetes dataclient has an empty string or wrong value and use the default if so.